src/sass/experimental/overpanel.scss

```diff
@@ -0,0 +1,53 @@
+/*
+_______   _______ ___________ ________  ________ _   _ _____ ___  _
+|  ___\ \ / / ___ \  ___| ___ \_   _|  \/  |  ___| \ | |_   _/ _ \| |
+| |__  \ V /| |_/ / |__ | |_/ / | | | .  . | |__ |  \| | | |/ /_\ \ |
+|  __| /   \|  __/|  __||    /  | | | |\/| |  __|| . ` | | ||  _  | |
+| |___/ /^\ \ |   | |___| |\ \ _| |_| |  | | |___| |\  | | || | | | |____
+\____/\/   \|_|   \____/\_| \_|\___/\_|  |_|____/\_| \_/ \_/\_| |_|_____/
+
+This is an EXPERIMENT and is not ready to be included in production!
+*/
+
+.overpanel {
+
+  .title-bar {
+    background-color: $header-background;
+    box-shadow: $header-shadow;
+
+    .title-content {
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+      max-width: $content-m;
+      margin: 0 auto;
+      padding: $space-m;
+      * { margin: 0; } //to overwrite global margin bottom
+      button { white-space: nowrap; }
+    }
+  }
+
+  .content {
+    margin-top: $space-xs; //to keep content from hiding shadow
+    max-width: $content-m;
+  }
+
+}
+
+/* Utility classes for max-width of Overpanel */
+.overpanel-s {
+  .title-content {
+    max-width: $content-s !important;
```

Comment:
Do we have to use `!important`?

Reply:
The way I structured the CSS file i needed important, but i can rearrange it to make that not needed it looks like! If I nest the `.overpanel-l`, `!important` is not needed.